integreat_cms/cms/templates/contacts/contact_list_row.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}
+<tr class="border-t border-solid border-gray-200 hover:bg-gray-100">
+    <td class="pl-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.title }}
+        </a>
+    </td>
+    <td>
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.name }}
+        </a>
+    </td>
+    <td>
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.poi }}
+        </a>
+    </td>
+    <td class="pr-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.email }}
+        </a>
+    </td>
+    <td class="pr-2">
+        <a href="{{ contact.website }}"
+           target="_blank"
+           rel="noopener noreferrer"
+           class="text-blue-500 hover:underline"
+           title="{% translate "To website" %}">
+            {{ contact.website }}
+        </a>
+    </td>
+    <td class="pl-2 pr-4 text-right min">
+        {% if perms.cms.edit_contact %}
```

Comment:
```suggestion
        {% if perms.cms.change_contact %}
```
Maybe remains from copy&paste?

Reply:
I think it was more that I was having the wrong thought here :)